build/testdata/051.build.golden

```diff
@@ -41,6 +41,13 @@ def function(
     elif y: print(y)
     else: print(z); print(y); print(x)
 
+    if x: print(x)
+
+    elif y: print(y)
+
+
+    else: print(z); print(y); print(x)
```

Comment:
Because the build mode for now uses PythonBlock nodes and prints them without any changes.

In the future the build mode will actually format def statements.

Reply:
If I remember correctly, it keeps only one empty line.

Reply:
Here the PythonBlock is the top-level def-block, whatever is inside is just printed as it is.

Reply:
Oh sorry, I didn't read correctly (diff was truncated after the `def`).